src/docker/DockerApi.ts

```diff
@@ -301,15 +298,14 @@ class DockerApi {
     ) {
         const self = this
 
-        const nameAndTag = imageNameIncludingTag.split(':')
-
-        const tag = nameAndTag[1] || 'latest'
-        const imageName = nameAndTag[0]
+        const { repository, tag } = dockerodeUtils.parseRepositoryTag(
+            imageNameIncludingTag
+        )
```

Comment:
I think you've lost the fallback to latest if it's empty

Reply:
Indeed! I just pushed a correction.